regler/src/main/kotlin/no/nav/pleiepengerbarn/uttak/regler/BeregnUtbetalingsgrader.kt

```diff
@@ -145,14 +162,18 @@ object BeregnUtbetalingsgrader {
     private fun beregnForAktivitetsGruppe(
         taptArbeidstidSomDekkes: Duration,
         arbeid: Map<Arbeidsforhold, ArbeidsforholdPeriodeInfo>,
-        fordeling: Map<Arbeidsforhold, Prosent>
+        fordeling: Map<Arbeidsforhold, Prosent>,
+        overstyrtInput: OverstyrtInput?
     ): UtbetalingsgraderOgGjenværendeTimerSomDekkes {
         val utbetalingsgrader = mutableMapOf<Arbeidsforhold, Utbetalingsgrad>()
         var sumTimerForbrukt = Duration.ZERO
         arbeid.forEach { (arbeidsforhold, info) ->
             val fordelingsprosent = fordeling[arbeidsforhold]
                 ?: throw IllegalStateException("Dette skal ikke skje. Finner ikke fordeling for $arbeidsforhold.")
-            if (info.jobberNormalt > Duration.ZERO) {
+
+            if (overstyrtInput != null && overstyrtInput.overstyrtUtbetalingsgradPerArbeidsforhold.any { it.arbeidsforhold == arbeidsforhold }) {
+                utbetalingsgrader[arbeidsforhold] = utledGradForOverstyrte(arbeidsforhold, info, overstyrtInput)
```

Comment:
For tilfeller hvor bare deler av aktivitetene er overstyrt, må utbetalingsgrader for resterende regnes på vanlig måte.